K8s/helm/values.yaml

```
@@ -22,8 +22,12 @@ eum:
 psp:
   enabled: false
 
-# For the mini ones minikube, minishift set to true
-nodeport: false
+
+web:
+  # autotrace-webhook binaries to log in debug level
+  debug_autotrace: false
```

```suggestion
  debugAutotrace: false
```
Ideally we'd maintain consistency with the rest of the config and not use snake case.